blocks/to-do/change-item-status.js

```diff
@@ -2,6 +2,12 @@ export default function changeItemStaus(event) {
   const listItem = event.target.closest('li');
   if (event.target.tagName === 'INPUT' && event.target.type === 'checkbox') {
     const completed = event.target.checked;
-    listItem.setAttribute('data-completed', completed);
+    let status;
+    if (completed) {
+      status = 'done';
+    } else {
+      status = 'open';
+    }
```

Comment:
let's use a one-liner here with a ternary operator. something like this would probably work

`const status = completed ? 'done' : 'open'`